src/runtime/auto-prompt-manager-test.js

```diff
@@ -162,346 +159,50 @@ describes.realWin('AutoPromptManager', (env) => {
-      );
-      // Legacy storage operations
-      storageMock
-        .expects('get')
-        .withExactArgs(StorageKeys.IMPRESSIONS, /* useLocalStorage */ true)
-        .resolves(null)
-        .once();
-      storageMock
-        .expects('set')
-        .withExactArgs(
-          StorageKeys.IMPRESSIONS,
-          sandbox.match.any,
-          /* useLocalStorage */ true
-        )
-        .resolves()
-        .exactly(1);
-      storageMock
-        .expects('get')
-        .withExactArgs(StorageKeys.DISMISSALS, /* useLocalStorage */ true)
-        .resolves(null)
-        .once();
-      storageMock
-        .expects('set')
-        .withExactArgs(
-          StorageKeys.DISMISSALS,
-          sandbox.match.any,
-          /* useLocalStorage */ true
-        )
-        .resolves()
-        .once();
-
-      await eventManagerCallback({
-        eventType: miniPromptEventType,
-        eventOriginator: EventOriginator.UNKNOWN_CLIENT,
-        isFromUserAction: null,
-        additionalParameters: null,
-      });
-      await eventManagerCallback({
-        eventType: largePromptEventType,
-        eventOriginator: EventOriginator.UNKNOWN_CLIENT,
-        isFromUserAction: null,
-        additionalParameters: null,
-      });
-      await eventManagerCallback({
-        eventType: dismissableEventType,
-        eventOriginator: EventOriginator.UNKNOWN_CLIENT,
-        isFromUserAction: null,
-        additionalParameters: null,
-      });
-    });
-  });
-
-  it('should locally store contribution dismissals when contribution dismissal events are fired', async () => {
-    storageMock
-      .expects('get')
-      .withExactArgs(StorageKeys.DISMISSALS, /* useLocalStorage */ true)
-      .resolves(null)
-      .once();
-    storageMock
-      .expects('set')
-      .withExactArgs(
-        StorageKeys.DISMISSALS,
-        CURRENT_TIME.toString(),
-        /* useLocalStorage */ true
-      )
-      .resolves()
-      .once();
+  it('should ignore undefined events', async () => {
+    autoPromptManager.isClosable_ = true;
+    storageMock.expects('get').never();
+    storageMock.expects('set').never();
 
     await eventManagerCallback({
-      eventType: AnalyticsEvent.ACTION_SWG_CONTRIBUTION_MINI_PROMPT_CLOSE,
+      eventType: undefined,
       eventOriginator: EventOriginator.UNKNOWN_CLIENT,
       isFromUserAction: null,
       additionalParameters: null,
     });
   });
 
-  it('should locally store subscription dismissals when subscription dismissal events are fired', async () => {
-    storageMock
-      .expects('get')
-      .withExactArgs(StorageKeys.DISMISSALS, /* useLocalStorage */ true)
-      .resolves(null)
-      .once();
-    storageMock
-      .expects('set')
-      .withExactArgs(
-        StorageKeys.DISMISSALS,
-        CURRENT_TIME.toString(),
-        /* useLocalStorage */ true
-      )
-      .resolves()
-      .once();
+  it('should not store event timestamps for a non frequency capping event', async () => {
```

Comment:
I really appreciate your thoroughness in test coverage! Would you be able to group relevant tests and put them under a `describe` block? Looks like some of them are already grouped so maybe we can just add one more to consolidate tests for timestamp logging. WDYT?

Reply:
Great idea! done!